scenes/gui.tscn

```diff
@@ -130,9 +130,9 @@ anchor_top = 0.5
 anchor_right = 0.5
 anchor_bottom = 0.5
 offset_left = -450.0
-offset_top = 130.0
+offset_top = 40.0
 offset_right = -82.0
-offset_bottom = 236.0
+offset_bottom = 146.0
```

Comment:
All the changes in this file are to adjust the pause menu to ensure it remains aligned.
Noticed misalignment when working on the sounds, now looks like this =)
<img width="634" alt="image" src="https://github.com/Gfecito/orball/assets/71032552/8ffe0f48-f576-4e04-b3ec-514182f6f120">